### DISCUSSION TITLE
Community moderators can HOLD and mark as spam, but current interface won't allow to use

### DISCUSSION THREAD

**Discussion by Kurdakov** [ThioJoe/YT-Spammer-Purge#157] [Ideas]
Community moderators can HOLD and mark messages as spam, but flag userNotChannelOwner prevents to use the feature in software for non channel owners, so I just modified my own version userNotChannelOwner = False
and use it for HOLD actions as a community moderator on one channel. 

Better yet would be to have somehow community moderator flag to make more granular feature availability.

**Comment by Kurdakov** [comment]
another  useful addition would be option to load videos from videos.txt and and spammers from spammers.txt, which could be edited separately before utility run, also possibly whitelist.txt with ids of commentators which might look like spammers (such that they help answer common questions - so their comments are frequently repeated), but should not be included into deletion process.

**Comment by ThioJoe** [comment]
Hm so if you have moderator permissions, you can confirm it actually lets you hold the comments and it works?
Edit: Just tried it myself and it works. I'll add it as a setting in the config file to override the 'not your channel mode'

**Reply by Kurdakov** [reply]
Thank you
